identity_ntp/recipes/default.rb

```diff
@@ -0,0 +1,20 @@
+package 'ntp' do
+  action :upgrade
+end
+
+template '/etc/ntp.conf' do
+  source 'ntp.conf.erb'
+  owner 'root'
+  group 'root'
+  mode 0644
+end
```

Comment:
Might be nice to add a `notifies :restart, 'service[ntp]'` here to be sure we're running the right config.